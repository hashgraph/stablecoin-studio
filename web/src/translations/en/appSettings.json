{
	"title": "Configure custom RPC/MIRROR",
	"subtitle": "Choose an option",
	"name": "Name",
	"rpc": "RPC",
	"mirrorNode": "Mirror node",
	"url": "Url",
	"apiKey": "Api Key",
	"addMirror": "Add new service",
	"addRPC": "Add RPC",
	"removeMirror": "Remove Mirror Node",
	"removeRpc": "Remove RPC",
	"header": "Header",
	"mirrorNameExists": "Mirror name already exists, if you want to update it remove it first and recreate it",
	"rpcNameExists": "RPC name already exists, if you want to update it remove it first and recreate it"
}
