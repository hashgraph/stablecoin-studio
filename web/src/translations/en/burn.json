{
    "title": "Burn tokens operation",
    "operationTitle": "Operation details",
    "amountLabel": "Enter the amount",
    "amountPlaceholder": "0",
    "originAccountPlaceholder": "0.0.0",
    "modalSuccessDesc": "{{amount}} tokens have been burned.",
    "modalAction": {
        "subtitle": "Operation details review",
        "originAccount": "Account",
        "amount": "Amount",
        "accept": "Burn tokens"
    }
}