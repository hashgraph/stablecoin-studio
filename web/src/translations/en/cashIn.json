{
    "title": "Cash in operation",
    "operationTitle": "Operation details",
    "amountLabel": "Introduce amount",
    "amountPlaceholder": "0",
    "destinationAccountLabel": "Destination account",
    "destinationAccountPlaceholder": "0.0.0",
    "modalSuccessDesc": "The cash in operation finished successfully.",
    "modalAction": {
        "subtitle": "Operation details review",
        "destinationAccount": "Destination account",
        "amount": "Amount",
        "accept": "Cash in tokens"
    }
}