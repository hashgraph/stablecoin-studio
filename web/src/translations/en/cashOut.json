{
    "title": "Cash out operation",
    "operationTitle": "Operation details",
    "amountLabel": "How much tokens?",
    "amountPlaceholder": "Introduce amount",
    "originAccountLabel": "From which account tokens will be withdrawn?",
    "originAccountPlaceholder": "Introduce account",
    "modalSuccessDesc": "The cash out operation finished successfully.",
    "modalAction": {
        "subtitle": "Operation details review",
        "originAccount": "Account to cash out",
        "amount": "Amount",
        "accept": "Make cash out"
    }
}