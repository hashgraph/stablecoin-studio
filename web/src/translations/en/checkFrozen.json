{
	"title": "Check account frozen status",
	"operationTitle": "Operation details",
	"accountLabel": "Enter the account",
	"accountPlaceholder": "0.0.0",
	"modalIsFrozen": "The account {{ account }} is frozen for the token.",
	"modalIsNotFrozen": "The account {{ account }} is unfrozen for the token.",
	"modalAction": {
		"subtitle": "Operation details review",
		"account": "Account",
		"accept": "Get account frozen status"
	}
}
