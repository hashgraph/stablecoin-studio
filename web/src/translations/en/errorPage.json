{
  "stableCoinNotSelected": {
    "title": "Choose a stable coin to start operating!",
    "description": "You have not chosen a stable coin yet.",
    "description2": "Choose an item to operate."
  }
}