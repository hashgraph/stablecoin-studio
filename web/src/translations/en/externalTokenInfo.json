{
    "common": {
      "nextStep" : "Next step",
      "goBack" : "Go back",
      "createStableCoin": "Create stable coin",
      "createNewStableCoin": "Import token",
      "cancel": "Cancel",
      "addExternalToken": "Import token"
    },
    "tabs": {
      "externalTokenInfo": "Token info"
    },
    "externalTokenInfo": {
      "stableCoinId": "Stable coin id",
      "stableCoinIdPlaceholder": "0.0.0",
      "title": "Imported token",
      "autoCheckRoles": "Do you want to get the roles automatically? This operation will incur a cost",
      "rolesPlaceholder": "Select the roles"
    },
    "optionalDetails": {
      "decimals": "Decimals",
      "expirationDate": "Expiration date",
      "finite": "Finite",
      "initialSupply": "Initial supply",
      "infinite": "Infinite",
      "placeholder": "Enter the {{placeholder}}",
      "maxSupply": "Max supply",
      "title": "Stable coin optional details",
      "typeSupply": "Type supply",
      "typeSupplyPlaceholder": "Choose type supply"
    },
    "managementPermissions": {
      "currentUserKey": "Current user key",
      "description": "Allow all the following features to be managed through smart contracts",
      "pause": "Pause",
      "freeze": "Freeze",
      "introduce": "Enter the {{name}}",
      "none": "None",
      "otherKey": "Other key",
      "placeholder": "Enter the {{placeholder}}",
      "rescue": "Rescue",
      "admin": "Admin",
      "supply": "Supply",
      "theSmartContract": "The smart contract",
      "title": "Stable coin management permissions",
      "totalSupply": "Total supply",
      "treasuryAccountAddress": "Treasury account address",
      "wipe": "Wipe"
    },
    "review": {
      "basicDetails": "Basic details",
      "managementPermissions": "Management permissions",
      "optionalDetails": "Optional details",
      "title": "Stable coin review"
    },
    "notification": {
      "title": "{{ result }} creating the stable coin",
      "descriptionSuccess": "The stable coin was created",
      "descriptionError": "Sorry, there was an error creating the stable coin"
    }
  }