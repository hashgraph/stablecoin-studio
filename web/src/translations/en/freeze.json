{
    "title": "Freeze operation",
    "operationTitle": "Operation details",
    "accountLabel": "Enter the account",
    "accountPlaceholder": "0.0.0",
    "modalSuccess": "The account {{ account }} is frozen.",
    "modalAction": {
        "subtitle": "Operation details review",
        "account": "Account",
        "accept": "Freeze account"
    }
}