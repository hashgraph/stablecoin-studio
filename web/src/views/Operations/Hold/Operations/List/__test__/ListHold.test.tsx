import { render } from '../../../../../../test';
import en from '../../../../../../translations/en/operations.json';
import { ListOperationHold } from '..';

const translations = en;

describe(`<${ListOperationHold.name} />`, () => {
	test('should render correctly', () => {
		const component = render(<ListOperationHold />);

		expect(component.asFragment()).toMatchSnapshot();
	});

	test('should render titles', () => {
		const component = render(<ListOperationHold />);

		expect(component.getByTestId('title')).toHaveTextContent(translations.hold.list.title);
	});
});
