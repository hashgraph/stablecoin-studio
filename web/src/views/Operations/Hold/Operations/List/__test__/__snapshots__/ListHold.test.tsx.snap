// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListOperationHold /> should render correctly 1`] = `
<DocumentFragment>
  <div
    class="chakra-stack css-16oka5y"
  >
    <h2
      class="chakra-heading css-bqsusc"
      data-testid="base-container-heading"
    >
      Operations
    </h2>
    <div
      class="css-1b6lda7"
      data-testid="base-container-children"
    >
      <div
        class="css-1us6gcn"
      >
        <div
          class="chakra-stack css-tap0v4"
        >
          <button
            class="chakra-button css-4mjrgu"
            data-testid="cancel-btn"
            type="button"
          >
            <svg
              class="chakra-icon css-6tzxjd"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"
                fill="currentColor"
              />
            </svg>
          </button>
          <h2
            class="chakra-heading css-1wns1s3"
            data-testid="title"
          >
            List holds
          </h2>
        </div>
        <div
          class="chakra-stack css-b5leqj"
        >
          <div
            class="chakra-table__container css-64celf"
          >
            <table
              class="chakra-table css-5605sr"
            >
              <thead
                class="css-iqkxbt"
              >
                <tr
                  class="css-0"
                >
                  <th
                    class="css-1zts0j"
                  >
                    ID
                  </th>
                  <th
                    class="css-1zts0j"
                  >
                    Amount
                  </th>
                  <th
                    class="css-1zts0j"
                  >
                    Expiration date
                  </th>
                  <th
                    class="css-1zts0j"
                  >
                    Escrow address
                  </th>
                  <th
                    class="css-1zts0j"
                  >
                    Destination address
                  </th>
                </tr>
              </thead>
              <tbody
                class="css-0"
              >
                <tr
                  class="css-0"
                >
                  <td
                    class="css-xumdn4"
                  >
                    <div
                      class="chakra-skeleton css-1jum603"
                    />
                  </td>
                  <td
                    class="css-xumdn4"
                  >
                    <div
                      class="chakra-skeleton css-1jum603"
                    />
                  </td>
                  <td
                    class="css-xumdn4"
                  >
                    <div
                      class="chakra-skeleton css-1jum603"
                    />
                  </td>
                  <td
                    class="css-xumdn4"
                  >
                    <div
                      class="chakra-skeleton css-1jum603"
                    />
                  </td>
                  <td
                    class="css-xumdn4"
                  >
                    <div
                      class="chakra-skeleton css-1jum603"
                    />
                  </td>
                </tr>
                <tr
                  class="css-0"
                >
                  <td
                    class="css-xumdn4"
                  >
                    <div
                      class="chakra-skeleton css-1jum603"
                    />
                  </td>
                  <td
                    class="css-xumdn4"
                  >
                    <div
                      class="chakra-skeleton css-1jum603"
                    />
                  </td>
                  <td
                    class="css-xumdn4"
                  >
                    <div
                      class="chakra-skeleton css-1jum603"
                    />
                  </td>
                  <td
                    class="css-xumdn4"
                  >
                    <div
                      class="chakra-skeleton css-1jum603"
                    />
                  </td>
                  <td
                    class="css-xumdn4"
                  >
                    <div
                      class="chakra-skeleton css-1jum603"
                    />
                  </td>
                </tr>
                <tr
                  class="css-0"
                >
                  <td
                    class="css-xumdn4"
                  >
                    <div
                      class="chakra-skeleton css-1jum603"
                    />
                  </td>
                  <td
                    class="css-xumdn4"
                  >
                    <div
                      class="chakra-skeleton css-1jum603"
                    />
                  </td>
                  <td
                    class="css-xumdn4"
                  >
                    <div
                      class="chakra-skeleton css-1jum603"
                    />
                  </td>
                  <td
                    class="css-xumdn4"
                  >
                    <div
                      class="chakra-skeleton css-1jum603"
                    />
                  </td>
                  <td
                    class="css-xumdn4"
                  >
                    <div
                      class="chakra-skeleton css-1jum603"
                    />
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</DocumentFragment>
`;
